### ISSUE TITLE
Toggle component does not render correctly if the label text is more than one line long

### ISSUE THREAD

**Issue by multiwebinc** [themesberg/flowbite#529]
Originally posted https://github.com/themesberg/flowbite-react/issues/587, however I believe this issue exists with flowbite, not just flowbite-react, because the CSS comes from here.

For example, https://flowbite.com/docs/forms/toggle/#toggle-example (I edited the label text with the dev tools):

![image](https://user-images.githubusercontent.com/901732/231971394-6c9c0c5a-5654-42ba-a4df-d86814adeaae.png)

The first issue is that the toggle switch background is no longer a pill shape, but instead a ball. The example has `w-11` for the toggle switch, which adds `width: 2.75rem;` to the CSS, but it also needs `min-width: 2.75rem;` to make it not appear like a ball when inside a flex container, however this does not appear possible with TailwindCSS. Maybe something other than flex needs to be used for the container? I'm not sure.

The second issue is that the ball is floating way up. That's because it is positioned absolutely with `after:top-[2px]`. I think the `:after` element needs to be positioned `relative`.

**Comment by multiwebinc** [comment]
How would anyone feel about a fix to this bug that requires the following alteration to the tailwind config file?

```js
module.exports = {
  theme: {
    extend: {
      minWidth: ({ theme }) => theme('width'),
    },
  },
}
```

**Comment by zoltanszogyenyi** [comment]
Fixed via https://github.com/themesberg/flowbite/commit/902e2ea084b0eccd6749d32779fdcdb9d1a0daa6. Will be released with `v2.3` soon. Thanks!
